.circleci/config.yml

```diff
@@ -1,39 +1,81 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
-version: 2
-jobs:
-  build:
-    docker:
-      # specify the version you desire here
-      - image: circleci/node:7.10
-      
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/mongo:3.4.4
+version: 2.1
 
-    working_directory: ~/repo
+workflows:
+  build-test:
+    jobs:
+      - prep-deps
+      - test-build:
```

Comment:
Could we omit this `test-build` step? The lint one makes sense because we will have a lint step in the near future, but I don't see a point in adding this if we'll never need a build.

Reply:
Removed